apps/demo/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "clean": "rimraf .turbo node_modules .next",
     "dev": "next dev -p 3002 --turbo",
-    "go": "next dev -p 3002 --turbo",
+    "go": "next dev -p 3002",
```

Comment:
Ohh yeah, there were some errors being thrown after the dev server started when I used the turbopack...This happened with some of my personal projects too (on my local machine), I suppose since turbopack is still in beta...